src/app/login-form/login-form.component.html

```diff
@@ -0,0 +1,25 @@
+<mat-card-title>Login</mat-card-title> <span id='invalidText' *ngIf="!hideInvalidText" [(ngModel)]="errorMessage" ngDefaultControl><h2>{{errorMessage}}</h2></span>
```

Comment:
This should be something like `<h2 mat-dialog-title>Login</h2>`, `mat-dialog-title` is a structural directive